tests/test_file.py

```diff
@@ -176,16 +176,11 @@ def test_local_path_load():
 
 
 def test_init_from_directory(get_test_filepaths):
-    isin_default_storage = get_test_filepaths[0]
```

Comment:
I'll need to add this back and define what the storage root for this path should be. Otherwise this operation isn't well-defined.